pkgs/servers/mastodon/source.nix

```diff
@@ -2,10 +2,10 @@
 { fetchgit, applyPatches }: let
   src = fetchgit {
     url = "https://github.com/tootsuite/mastodon.git";
-    rev = "v3.3.0";
-    sha256 = "17wvggvy5mmyf3f1i5v1hgvh6wjdhg9hb3wiyfaydx0slsg03qba";
+    rev = "v3.4.0";
```

Comment:
Please use fetchFromGitHub.

Reply:
Currently the updateScript has an `--url` flag, which takes an arbitrary git url, e.g. for a custom fork of Mastodon. I'd rather keep that functionality than narrowing it down to GitHub only repos. Also distinguihsing from the URL wether to use `fetchFromGithub` or `fetchgit` sounds tedious.

So I'd vouch for keeping it that way.

Reply:
Using fetchFromGitHub is way faster because it can download a tarball instead of git cloning. Also you can just overwrite src with anything to track a fork which is way easier.